activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -18,7 +18,9 @@ module FinderMethods
     #   Person.find([1])        # returns an array for the object with ID = 1
     #   Person.where("administrator = 1").order("created_on DESC").find(1)
     #
-    # NOTE: The returned records are in the same order as the ids you provide.
+    # NOTE: The returned records are in the same order as the ids you provide,
+    # but only if there is no order on the scope.
```

Comment:
What do you think about inverting the wording a bit, e.g.
```suggestion
    # NOTE: The returned records respect ordering on the original scope.
    Otherwise, the returned records are in the same order as the ids provided.
```
Just so that it's a bit clearer that scope is preserved, and _then_ we take into account the ids that were passed in.